Project: To Do List

lucasmazo32 opened the pull request:
I hope you enjoy the project!

Features: Every changed or added items go into the first table. The reason is because this table is a FIFO storage. Therefore no order is imposed.

After refreshing the page, all added or changed Items go to their project in the correct order (by date).  This only happens once because otherwise, the changes in the DOM and the backend will be more substantial than necessary.

adriaanbd commented on src/todoItem.js:
```diff
@@ -0,0 +1,51 @@
+// list default value creation
+const listStored = localData[1];
+let list = {};
+if (localData[1] !== null) {
```
A variable has been created on line  `8` for this, please use this instead [`OPTIONAL`].

lucasmazo32 commented:
As explained in the beginning, you needed to refresh the page to see the organized version of the to do's in their project. I added a button "Organize projects" when you add or change a to-do item so the user can click it and, thus, see all the elements organized by date in their respective project container. The mentioned button will only appear if the user adds or changes at least one project, so it will not confuse more if its there.

Added the server for development mode so now you can start it with ``` npm run start:dev ```.

Thanks!
